### DISCUSSION TITLE
new macro/pragma: @graphql

### DISCUSSION THREAD

**Discussion by brianleroux** [architect/architect#1256] [Feature Requests]
Never liked AppSync for the VTL resolver thing. But they fixed that part! We should get support in ASAP via macro.

https://aws.amazon.com/blogs/mobile/appsync-direct-lambda

**Comment by trsrm** [comment]
Any updates on this?

**Reply by brianleroux** [reply]
still want to do it…most devs use Apollo in a Lambda function so not a huge drive for the functionality; are you using AppSync today?

**Reply by trsrm** [reply]
Just looking to test it.

**Reply by brianleroux** [reply]
do you use graphql at all or just kicking tires ?

**Reply by trsrm** [reply]
I'm playing with GraphQL + Lambda + other serverless stuff. Successfully deployed Apollo to Lambda, and going try AppSync. Off course, I can setup everything manually, but just wanted to check your progress on this...
